app/services/organization_templates.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class OrganizationTemplates
+  def self.all
+    @all || Dir.glob("lib/templates/*.yml").map do |file|
```

Comment:
```suggestion
    @all || Dir.glob("#{Rails.root.join(Rails.application.secrets.affac.templates_folder}/*.yml").map do |file|
```
Let's use a key in config/secrets.yml like this
```yaml
default:
  affac:
    templates_folder: lib/templates
```
The in the test section we can use the folder
```yaml
affac:
  templates_folder: spec/fixtures/templates
```
So we can have the test templates there and only the good ones in the other folder

Reply:
Done. Thanks